advanced_functionality/autogluon-tabular/AutoGluon_Tabular_SageMaker.ipynb

```diff
@@ -595,6 +603,165 @@
    "source": [
     "predictor.delete_endpoint()"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Explainability with Amazon SageMaker Clarify\n",
+    "\n",
+    "There are expanding business needs and legislative regulations that require explainations of why a model mades the decision it did. SHAP (SHapley Additive exPlanations) is an approach to explain the output of machine learning models. SHAP values represent a feature's contribution to a change in the model output. SageMaker Clarify uses SHAP to explain the contribution that each input feature makes to the final decision."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "##### Set parameters for SHAP calculation"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "seed = 0\n",
+    "num_rows = 500\n",
+    "\n",
+    "#Write a csv file used by SageMaker Clarify\n",
+    "test_explainavility_file = 'test_explainavility.csv'\n",
+    "train.head(num_rows).to_csv(test_explainavility_file, index=False, header=False)\n",
```

Comment:
I'd rather explain some of the test data than train data (here and below). Also it'd be worthwhile to create a variable that you can refer to in the shap plot below.